openfisca_uk/parameters/calibration/programs/total_NI/revenue/in_total.yaml

```diff
@@ -0,0 +1,15 @@
+total_NI:
```

Comment:
could you spell out as `national_insurance`?

Reply:
Oh here the names are the same as the OpenFisca-UK variable names: `national_insurance` is just employee-side, whereas `total_NI` includes employer contributions. Should we rename them?

Reply:
Ah, yes I think we should but doesn't have to be here